website/blog/2017-12-18-jest-22.md

```diff
@@ -101,7 +102,14 @@ test('my mocking test', () => {
       },
     ],
   ],
+  "results": Array [
```

Comment:
did we have this in jest 22?

Reply:
Good catch. Just looked at the code, it was not there indeed.